schemas/metadata-events/AddData.json

```diff
@@ -4,9 +4,7 @@
   "description": "Indicates that data has been ingested into a root dataset.",
   "type": "object",
   "additionalProperties": false,
-  "required": [
-    "outputData"
-  ],
+  "required": [],
```

Comment:
Removing `outputData` from required fields likely impacts our web UI, should be tested.